client/src/components/App/index.js

```diff
@@ -1,12 +1,94 @@
 import React from 'react';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Redirect,
+} from 'react-router-dom';
+import axios from 'axios';
+
 import './index.css';
+import AppBar from '../common/AppBar';
+import Limitation from '../common/limitation';
```

Comment:
transfer these lines above style file , and leave line then import the style file